src/generators/web.cr

```diff
@@ -10,8 +10,11 @@ class LuckyCli::Generators::Web
     project_name : String,
     @options : Options
   )
-    @project_dir = project_name.gsub(" ", "_")
-    @project_name = @project_dir.gsub("-", "_")
+    @project_dir = project_name
+    @project_name = project_name
```

Comment:
It looks like the sanitizer/validator allows dashes (which is a expected), but the dashes should be changed to "_". Otherwise the shard name and generate file will have a dash and will break deployment on Heroku because it will not be able to find the file (it'll be looking for one with an underscore). I think using what was there previously would fix it. `@project_name = @project_dir.gsub("-", "_")`